Library/Formula/contacts-phonetic.rb

```diff
@@ -0,0 +1,12 @@
+class ContactsPhonetic < Formula
+  desc "Add phonetic names to contacts."
+  homepage "https://github.com/Elethom/ContactsUtil-PhoneticName"
+  url "https://github.com/Elethom/ContactsUtil-PhoneticName/archive/7f13963b6948e27dca4ae06ba9939ffe26cf0089.tar.gz"
+  version "0.1.1"
+  sha256 "3816152a62674870489b5780aa10b6bdb5adff9e55f79c19b6529a833859e15b"
+
+  def install
+    system "make"
+    bin.install "contacts-phonetic"
```

Comment:
@cosmo0920 Thanks for your suggestion. Well, honestly I've got no idea at all. I'm more of a designer than engineer. This is my first time writing a `Makefile` and Homebrew formula ever. Spent only half a day trying to learn that. All I could know is that my change sort of... worked. :sob: